src/test/java/gameboy/gamepack/racinggame/data/vo/PositionTest.java

```diff
@@ -0,0 +1,33 @@
+package gameboy.gamepack.racinggame.data.vo;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class PositionTest {
+
+    @Test
+    @DisplayName("초기 Position의 add 결과 테스트")
+    void add() {
+        //given
+        Position position = new Position();
+        //when
+        position.add();
+        //then
+        assertThat(position.getPosition()).isEqualTo(1);
```

Comment:
자동차 위치를 나타내는 값 객체를 만드셨네요
그러면 이렇게도 비교해볼 수 있을 것 같아요

```suggestion
        assertThat(position).isEqualTo(new Position(1));
```
equals를 구현하면서 hashcode도 같이 만들어두면 좋을 것 같습니다.

Reply:
지금은 Position의 noArgs 생성자밖에 없는데 테스트 코드를 위해 생성자를 추가하는건 상관없나요??

Reply:
이것도 항상 호불호가 갈리는 내용인데요
저는 개인적으로 필요하다면 생성할 수 있다고 생각합니다.

지훈님이 Position도 값이라고 판단해서 VO로 만들었을거에요 
그렇기 때문에 값은 언제든지 생성할 수 있다고 생각합니다.

그래도 외부에서 호출되는 게 좀 찝찝하다면 
default 접근 제한자를 사용해서 같은 패키지에서만 호출될 수 있는 생성자로 만들면 되지 않을까 싶네요
테스트 코드도 보통 같은 패키지에 생성하니까 괜찮을 것 같고요

```java
    public Position() {  }

    Position(int position) {
        this.position = position;
    }
```